### PR TITLE
refactor(can): make motor message handler execute actual motor control functions

### PR DESCRIPTION
**Pull request by Laura-Danielle**
## Overview
With this PR, you should be able to send CAN messages to execute actual motor control. As noted with the TODO message, there is definitely some refactors we can do around templating the motor control stuff.

@amitlissack whenever you make it to the office next, I would appreciate us trying to test this together.

### CODE REVIEW

**Review by sfoster1 — APPROVED** [review]
Nice

File: include/can/core/messages.hpp [hunk 1]
```diff
@@ -113,22 +113,18 @@ struct GetStatusResponse : BaseMessage<MessageId::get_status_response> {
 };
 
 struct MoveRequest : BaseMessage<MessageId::move_request> {
-    uint32_t distance;
-    uint32_t speed;
+    uint32_t steps;
```
Comment (amitlissack) [inline include/can/core/messages.hpp:116]: Hold on here. I thought we wanted the can message to contain speed and distance?

**Review by amitlissack — CHANGES_REQUESTED** [review]
I don't get why the can move request was changed.

File: include/can/core/messages.hpp (hunk 1 above)
Comment (Laura-Danielle) [reply include/can/core/messages.hpp:116]: I changed it for the demo. If we want to keep it as we expect it to be in the future, that's fine but we'll need to use UART for the demo then.

Comment (amitlissack) [reply include/can/core/messages.hpp:116]: Understood.

You'll have to fix some unit tests I believe.

**Review by amitlissack — APPROVED** [review]

File: include/can/core/messages.hpp (hunk 1 above)
Comment (Laura-Danielle) [reply include/can/core/messages.hpp:116]: yep I'm in the midst of doing that now.

**Review by ahiuchingau — COMMENTED** [review]
cool!

**Pull request merged**